test/test-component.js

```diff
@@ -110,6 +110,15 @@ describe('Simple Component instance', function() {
       expect(el.shadowRoot).to.be.ok;
     });
 
+    it('successfully finds the panel root', function() {
+      el.attachedCallback();
```

Comment:
`el.attachedCallback` should already have been run by now because of the `beforeEach`. maybe we should enforce non-reentrance